node/chainSimulator/components/bootstrapComponents.go

```diff
@@ -38,7 +38,7 @@ type bootstrapComponentsHolder struct {
 }
 
 // CreateBootstrapComponentHolder will create a new instance of bootstrap components holder
-func CreateBootstrapComponentHolder(args ArgsBootstrapComponentsHolder) (factory.BootstrapComponentsHolder, error) {
+func CreateBootstrapComponentHolder(args ArgsBootstrapComponentsHolder) (factory.BootstrapComponentsHandler, error) {
```

Comment:
Please rename to `CreateBootstrapComponents` and remove the inner `closeHandler` declaration & usage

Reply:
renamed